packages/modules/renault/api.py

```diff
@@ -0,0 +1,218 @@
+    dmp['response'] = str(response)
+    dmp['responsetext'] = str(responsetext)
+    dump_json(dmp, "soc_renault_4_reply_vh" + str(vehicle))
+
+    # Step 5
+    gigya_jwt = json.loads(responsetext)
+    gigya_jwttoken = gigya_jwt['id_token']
+    payload = {'country': country}
+    data = urllib.parse.urlencode(payload)
+    data = data.encode('Big5')
+    reg = urllib.request.Request(kamereonrooturl + '/commerce/v1/persons/' +
+                                 kamereonpersonid + '?' + data.decode("utf-8"))
+    reg.add_header('x-gigya-id_token', gigya_jwttoken)
+    reg.add_header('apikey', kamereonapikey)
+    dmp = {}
+    dmp['data'] = data.decode("utf-8")
+    dmp['reg'] = str(reg)
+    dump_json(dmp, "soc_renault_5_request_vh" + str(vehicle))
+    response = urllib.request.urlopen(reg)
+    responsetext = response.read()
+    dmp = {}
+    dmp['response'] = str(response)
+    dmp['responsetext'] = str(responsetext)
+    dump_json(dmp, "soc_renault_5_reply_vh" + str(vehicle))
+    kamereon_per = json.loads(responsetext)
+    kamereonaccountid = kamereon_per['accounts'][0]['accountId']
+
+    # Step 6 - skipped
+    # print(time_string,'kamereonaccountid',kamereonaccountid)
+    # payload = {'country': country}
+    # headers = {'x-gigya-id_token': gigya_jwttoken, 'apikey': kamereonapikey}
+    # data = urllib.parse.urlencode(payload)
+    # data = data.encode('Big5')
+    # reg = urllib.request.Request(kamereonrooturl + '/commerce/v1/accounts/'
+    # + kamereonaccountid + '/kamereon/token?'  + data.decode("utf-8"))
+    # reg.add_header('x-gigya-id_token',gigya_jwttoken)
+    # reg.add_header('apikey', kamereonapikey)
+    # print('c6',reg)
+    # response= urllib.request.urlopen(reg)
+    # responsetext  = response.read()
+    # dmp = {}
+    # dmp['response'] = str(response)
+    # dmp['responsetext'] = str(responsetext)
+    # dump_json(dmp, "soc_renault_6_reply_vh" + str(vehicle))
+    # kamereon_token = json.loads(responsetext)
+    # kamereonaccesstoken = kamereon_token['accessToken']
+    # print(time_string,'kamereonaccesstoken',kamereonaccesstoken)
+
+    # Step 7 - vehicles
+    payload = {'country': country}
+    data = urllib.parse.urlencode(payload)
+    data = data.encode('Big5')
+    reg = urllib.request.Request(kamereonrooturl + '/commerce/v1/accounts/' +
+                                 kamereonaccountid + '/vehicles?' + data.decode("utf-8"))
+    reg.add_header('x-gigya-id_token', gigya_jwttoken)
+    reg.add_header('apikey', kamereonapikey)
+    dmp = {}
+    dmp['payload'] = str(payload)
+    dmp['data'] = data.decode("utf-8")
+    dmp['reg'] = str(reg)
+    dump_json(dmp, "soc_renault_7_request_vh" + str(vehicle))
+    response = urllib.request.urlopen(reg)
+    responsetext = response.read()
+    vehic = json.loads(responsetext)
+    dmp = {}
+    dmp['response'] = str(response)
+    dmp['responsetext'] = str(responsetext)
+    dmp['vehic'] = str(vehic)
+    dump_json(dmp, "soc_renault_7_reply_vh" + str(vehicle))
```

Comment:
```suggestion
    payload = {'country': country}
    vehic=req.get_http_session().post(f"{kamereonrooturl}/commerce/v1/accounts/{kamereonaccountid}/vehicles", headers={'x-gigya-id_token': gigya_jwttoken,
     'apikey': kamereonapikey}, data=json.dumps(payload)).json()
```
Der Request sieht dann so aus (ich kann es nicht testen, evtl muss zum Post noch ein `verify=False` hinzu)

Reply:
Die Änderung ist bestimmt sinnvoll. und ich würde es übernehmen, wenn ich könnte.
Ich kann es aber leider nicht mehr testen; 
Jarry hatte mich gebeten den Renault-Modul zu migrieren und dazu seinen myRenault-Account nach Passwort-Änderung geteilt.
Nachdem es lief, ist das Passwort wieder geändert.

Solche substantiellen Änderungen sollte auch jemand machen, der dauerhaften Zugriff auf den entsprechende Fahrzeug-Account hat.

Also erst mal "as-is" mergen oder es bleibt hat ein PR bis sich jemand findet.